requirements.txt

```diff
@@ -18,8 +18,6 @@ annotated-types==0.7.0
     # via pydantic
 anyio==4.10.0
     # via httpx
-appnope==0.1.4
```

Comment:
I guess the random differences here come from either updating the deps on a mac or on a linux?